Chummer/sheets/Notes set.xslt

```diff
@@ -238,6 +215,191 @@
           </xsl:if>
         </xsl:if>
 
+        <xsl:variable name="martialartslist" select="martialarts/martialart[notes != ''] or martialarts/martialart/martialarttechniques/martialarttechnique[notes != '']"/>
+        <xsl:if test="$martialartslist">
+          <div class="block" id="MartialArtsBlock">
+            <table><tr><td/></tr></table>
+            <xsl:call-template name="TableTitle">
+              <xsl:with-param name="name" select="$lang.MartialArts"/>
+            </xsl:call-template>
+            <table class="tablestyle">
+              <xsl:call-template name="MartialArtsList"/>
+            </table>
+          </div>
+          <xsl:call-template name="RowSummary">
+            <xsl:with-param name="text" select="$lang.MartialArts"/>
+            <xsl:with-param name="blockname" select="'MartialArtsBlock'"/>
+          </xsl:call-template>
+        </xsl:if>
+
+        <xsl:variable name="cyberlist" select="cyberwares/cyberware[notes != ''] or cyberwares/cyberware/children/cyberware[notes != '']"/>
```

Comment:
Keep in mind, cyberware can theoretically nest an infinite number of times. You want a selection that gets every single descendant, not just the first children. This is most obvious when you start dealing with modular cyberware.